packages/react-notifications/src/components/notification/notification.styles.ts

```diff
@@ -60,23 +76,30 @@ const getStylesForIcon = (props: StyleProps) => css`
   top: 0;
   width: 48px;
   height: 100%;
-  color: ${customProperties.colorSurface};
+  color: ${designTokens.colorSurface};
   border-radius: 3px 0 0 3px;
   background: ${getColorByType(props.type)};
 `;
 
+const getStylesForCloseIcon = (isNewTheme = false) => css`
+  ${isNewTheme ? '& svg { width: 12px; height: 12px; }' : ''}
+`;
+
 const getStylesForContent = (props: StyleProps) => {
   const fontColor =
     props.domain === NOTIFICATION_DOMAINS.SIDE
-      ? customProperties.colorSolid
-      : customProperties.colorSurface;
+      ? designTokens.colorSolid
+      : designTokens.colorSurface;
   return css`
     flex-basis: 100%;
     flex-grow: 1;
-    padding: 0 ${customProperties.spacingS};
+    padding: 0
+      ${props.isNewTheme ? designTokens.spacingM : designTokens.spacingS};
     margin: 0;
     font-size: ${props.domain === NOTIFICATION_DOMAINS.SIDE
-      ? '0.929rem'
+      ? props.isNewTheme
+        ? '1rem'
+        : '0.929rem'
```

Comment:
I wasn't sure about importing the `application-components` package in this one just for this two variants.

Please let me know what you think.

Reply:
In the end it's gonna be 1rem after we remove the old theme. So I think it's fine like this.